src/transformers/models/efficientloftr/configuration_efficientloftr.py

```diff
@@ -143,6 +143,11 @@ def __init__(
         self.stage_num_blocks = stage_num_blocks if stage_num_blocks is not None else [1, 2, 4, 14]
         self.stage_hidden_expansion = stage_hidden_expansion if stage_hidden_expansion is not None else [1, 1, 1, 1]
         self.stage_stride = stage_stride if stage_stride is not None else [2, 1, 2, 2]
+        self.stage_out_channels = [
+            self.stage_block_dims[i] * self.stage_hidden_expansion[i] for i in range(len(self.stage_block_dims))
+        ]
```

Comment:
can we avoid having thee of them?
- stage_out_channels
- stage_hidden_expansion
- stage_block_dims

I mean we can only have exact values of `stage_out_channels` and remove stage_hidden_expansion and stage_block_dims, right?

Reply:
commented: https://github.com/huggingface/transformers/commit/71fb8c2bfbd50ceb624a2cad2817df49e63b35d3#r162215426

Reply:
Thanks!